app/controllers/playbooks_controller.rb

```diff
@@ -56,7 +56,7 @@ def update
   # DELETE /playbooks/1
   # DELETE /playbooks/1.json
   def destroy
-    @playbook.destroy
+    @playbook.archive!
```

Comment:
This is our new 'archive!' method that adds a datetime value to the new column archived_at.